pkg/frontend/frontend.go

```diff
@@ -59,4 +74,27 @@ func NewServer(jsonRPC server.JSONRPC) *Server {
 			ScsiLuns:  make(map[string]*pb.VirtioScsiLun),
 		},
 	}
+
+	for _, option := range options {
+		server = option(server)
```

Comment:
why create a new server `server = option(server)`  here ? is it to chain calls ? like `optionA(optionB(server))` ? If so, need to document this